Netflix Clone/Views/SplashView.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  SplashView.swift
+//  Netflix Clone
+//
+//  Created by Polina Poznyak on 27.06.23.
+//
+
+import SnapKit
+import UIKit
+
+class SplashView : UIView {
+    // MARK: - Ini
+    init() {
+        super.init(frame: .zero)
+        initialize()
+    }
+    
+    //нужен, если хотим инициализировать view не из кода, а из сториборда
```

Comment:
It's better to use English comments ;) 
Just in case somebody else from the company decides to take a look at your repo